scripts/build_powerplants.py

```diff
@@ -294,6 +296,12 @@ def replace_natural_gas_fueltype(df):
         config["main_query"] = ppl_query
     else:
         config["main_query"] = ""
+    try:
+        check_output("java -version", stderr=STDOUT, shell=True).decode("utf-8")
+    except OSError:
+        logger.critical(
+            "java not found on path. You need to install openjdk to run build_powerplants.py"
+        )
```

Comment:
Thank @Tomkourou!

I think we could encapsulate this into a function, like `def test_java_availability():` and place these lines as a content.

Then the call `test_java_availability()` can be placed  right at the beginning of main at around line 270.

I think we are close to finalize :)